language/stdlib/modules/DesignatedDealer.move

```diff
@@ -225,12 +280,34 @@ module DesignatedDealer {
                 dealer.window_inflow == old(dealer.window_inflow) + amount);
         ensures tier_index < len(old(dealer).tiers);
         ensures dealer.window_inflow <= old(dealer).tiers[tier_index];
+
+        include TieredMint<CoinType>;
+    }
+
+    spec schema TieredMint<CoinType> {
```

Comment:
Canonical name would be `TieredMintAbortsIf`.

Reply:
Done.